config.yaml.defaults

```diff
@@ -75,6 +75,72 @@ app:
       icon: Info
       url: /about
 
+  homepage_widgets:
+    # Layouts arrays should be [x, y, w, h]
+    # See https://github.com/STRML/react-grid-layout
+
+    # component_props are properties passed down to the
+    # underlying React component, indicated by component_name
+
+    sourceCounts:
+      show: true
+      component_name: SourceCounts
```

Comment:
Yeah, it was mainly a convenience thing because I couldn't find a quick and easy way to capitalize the first letter in the template to get to the component name from the section name (the builtin capitalize filter also makes the rest of the string lowercase). But in retrospect, the section names are only used as CSS id values so they could easily be capitalized in the configs to get around that.